.github/workflows/diff.yml

```diff
@@ -0,0 +1,125 @@
+name: Package Differences
+
+on:
+  schedule:
+    - cron: "0 5 * * *"
+
+env:
+  OS_IMAGE_v1_3_x: registry.opensuse.org/isv/rancher/harvester/os/v1.3/main/baseos:latest
+  OS_IMAGE_v1_2_x: registry.opensuse.org/isv/rancher/harvester/os/v1.2/main/baseos:latest
+
+jobs:
+  get-all-releases:
+    runs-on: ubuntu-latest
+    outputs:
+      releaseName_v1_3_x: ${{ steps.get_release_v1_3_x.outputs.releaseName }}
+      releaseName_v1_2_x: ${{ steps.get_release_v1_2_x.outputs.releaseName }}
+
+    steps:
+      - name: Get all releases
+        id: get_all_releases
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const response = await github.rest.repos.listReleases({
+            owner: context.repo.owner,
+            repo: context.repo.repo
+            });
+            
+            const releases = response.data;
+            core.setOutput('allReleases', releases);
+      - name: Get v1.3 latest release
+        id: get_release_v1_3_x
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const releases = ${{ steps.get_all_releases.outputs.allReleases }};
+            const releaseNameToFind = "v1.3";
+            const foundRelease = releases.find(release => release.name.includes(releaseNameToFind));
```

Comment:
Can we use `startsWith` to filter releases

Reply:
good suggestion, updated!